nncf/tensorflow/__init__.py

```diff
@@ -18,14 +18,20 @@
 import tensorflow
 from pkg_resources import parse_version
 
-tensorflow_version = parse_version(tensorflow.__version__).base_version
+try:
+    _tf_version = tensorflow.__version__
+    tensorflow_version = parse_version(_tf_version).base_version
+except:
+    nncf_logger.debug("Could not parse tensorflow version")
+    _tf_version = '0.0.0'
+    tensorflow_version = parse_version(_tf_version).base_version
```

Comment:
`WARNING:nncf:NNCF provides best results with tensorflow==2.11.*, while current tensorflow version is 0.0.0. If you encounter issues, consider switching to tensorflow==2.11.*`

We only really need this for when the tensorflow module is mocked externally by the documentation collector and therefore has no version set. User-side, this could only get triggered if tensorflow package is not installed, and in this case this file would produce an exception way above at line 18, so this code won't be triggered.

Reply:
Honestly, version 0.0.0 in the warning message confuses me. I would prefer more relevant message like this "Could not parse tensorflow version, please, reinstall tensorflow"